forch/port_state_manager.py

```diff
@@ -104,15 +105,22 @@ def _handle_infracted_state(self):
 
 class PortStateManager:
     """Manages the states of the access ports for orchestrated testing"""
+
+    # pylint: disable=too-many-arguments
     def __init__(self, process_device_placement, process_device_behavior, get_vlan_from_segment,
                  update_device_state_varz=None, testing_segment=None):
         self._state_machines = {}
         self._static_port_behaviors = {}
         self._static_device_behaviors = {}
         self._dynamic_device_behaviors = {}
-        self._process_device_placement = process_device_placement
-        self._process_device_behavior = process_device_behavior
-        self._get_vlan_from_segment = get_vlan_from_segment
+        self._process_device_placement = (
+            lambda *args, **kwargs: process_device_placement(*args, **kwargs)
```

Comment:
why are you doing it this way?  This seems very fragile -- any change to the arguments will then implicitly effect all these functions. This really doesn't feel right (not sure what it's doing, but...). You should explicitly use the arguments that need to be used unless you're doing something that's very "generic" -- typically any use of *args or **kwargs is a bad idea...

Reply:
I think previsouly the number of args passed to the lambda function may differ in different cases. I have udpated it so that it does use args and kwargs. PTAL...